src/App.vue

```diff
@@ -43,7 +43,7 @@ import { RouterLink, RouterView } from 'vue-router'
       </div>
       </div>
       
-      <div class="container shadow-none border-color-none">
+      <div class=" shadow-none border-color-none">
```

Comment:
remove extra white spaces
```suggestion
      <div class="shadow-none border-color-none">
```